solutions/devsprint-eduardo-bocato-1/SpaceApp/Modules/Core/UI/Components/Views/LoadingView.swift

```diff
@@ -0,0 +1,24 @@
+import SwiftUI
+
+struct LoadingView: View {
+    var message: String?
+    
+    var body: some View {
+        VStack {
+            ProgressView()
```

Comment:
Also, the view is supposed to center itself on the screen... We are missing this here.